Open_Foam/Simulation.ipynb

```diff
@@ -5,199 +5,233 @@
-    "\\int \\rho \\mathbf{g}~dV = \n",
-    "\\frac{\\partial}{\\partial t} \\int \\rho \\mathbf{V} dV +\n",
-    "\\int (\\mathbf{n}~.~\\rho \\mathbf{V}) \\mathbf{V} dA .\n",
-    "$$\n",
+    "```C++\n",
+    "application     pimpleFoam;\n",
     "\n",
+    "startFrom       latestTime;\n",
     "\n",
-    "## Cite Source(s)\n",
+    "startTime       0;\n",
+    "\n",
+    "stopAt          endTime;\n",
+    "\n",
+    "endTime         150;\n",
+    "\n",
+    "deltaT          0.01;\n",
+    "\n",
+    "writeControl    runTime;\n",
+    "\n",
+    "writeInterval   0.5;\n",
     "```\n",
-    "> For more details on the physics, theory and equation derivation please see chapter 9, section 9.2 of \"A Physical Introduction to Fluid Mechanics by Alexander J. Smits\" 2nd edition. [Download Book Here!](http://www.efluids.com/efluids/books/efluids_books.htm)\n",
+    "For this problem 150 seconds of run time is set with time steps of 0.01 seconds. The outputs will be written every 0.5 second till the end of the run time. It should be noted that these are the corresponding settings for case study with Reynolds number of 100 and Tip Speed Ratio ($TSR=\\frac{R*\\omega}{U_0}$) of 1, where time step size of 0.01 s is small enough to correctly capture the flow fields in the domain. Users can change the length of the time interval and time steps according to the physics of the flow field and numerical simulation requirements.  \n",
+    "\n",
+    "Before start running simulation one additional step for simulation setup should be taken. This is the step of setting up drag force coefficient calculation on the cylinder and reporting it at every single iteration. As discussed earlier in the theory section according to dimensional analysis of flow over a cylinder, Reynolds number and drag force coefficients are the two non-dimensional variables that describe the physics of this problem. Therefore, monitoring the convergence trend of this these two variables is vital. To calculate and report the drag force coefficient for flows with different Reynolds number in every single iteration, user can take advantage of pre-defined functions and libraries developed in OpenFOAM. One can simply add the following force calculation function into the end of the `controlDict` dictionary file located in `\\system` directory as follows:\n",
+    "\n",
+    "```C++\n",
+    "functions\n",
+    "{\n",
+    "\n",
+    "    forcesCoeffs\n",
+    "    {\n",
+    "\t    type        forceCoeffs;\n",
+    "\n",
+    "\t    functionObjectLibs ( \"libforces.so\" );\n",
+    "\n",
+    "\t    outputControl   timeStep;\n",
+    "\t    timeInterval    10;\n",
+    "\n",
+    "\t    log         no;\n",
+    "\n",
+    "\t    patches     ( cylinder );\n",
+    "\t    pName       p;\n",
+    "\t    UName       U;\n",
+    "\t    rhoName     rhoInf;  // Indicates incompressible\n",
+    "\t    log         true;\n",
+    "\t    rhoInf      100;     // Redundant for incompressible\n",
```

Comment:
Great! This was elaborated at the corresponding section and density reference value was set to 1 for sake of consistency with other variables.
This pull-request will then be officially merged to the master branch now.